core/tests/org.openjdk.jmc.flightrecorder.configuration.test/src/test/java/org/openjdk/jmc/flightrecorder/configuration/model/xml/XMLTagInstanceTest.java

```diff
@@ -30,7 +30,7 @@
  * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
  * WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.openjdk.jmc.flightrecorder.controlpanel.ui.configuration.model.xml;
```

Comment:
Better move it to package org.openjdk.jmc.flightrecorder.configuration.model.xml.test.